src/styles/global.css

```
@@ -32,59 +29,74 @@ html[data-theme-ready='true'] body {
 h3,
-h4,
+h4 {
+	font-family: inherit;
+	font-weight: 700;
+	letter-spacing: -0.03em;
+	margin-top: 0;
+}
+
 .mono {
-	font-family: 'IBM Plex Mono', monospace;
+	font-family: ui-monospace, 'SFMono-Regular', 'SF Mono', Consolas, 'Liberation Mono', monospace;
```

Stylelint (17.7.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)